plugins/DuskVerb/tools/tuner/dense32_kurtosis_sweep.py

```
@@ -0,0 +1,178 @@
+    """offsets[i] in [-WIN[i], WIN[i]] -> snapped, distinct, sorted 32 primes.
+    Collisions fall back to the nearest unused prime in EITHER direction, so one
+    near the KMAX ceiling steps downward instead of overshooting the clamp."""
+    used = set(); out = []
+    for i, off in enumerate(offsets):
+        p = _nearest_unused(ANCHORS[i] + off, used)
+        used.add(p); out.append(p)
+    return sorted(out)
+
+
+def render_kurts(delays, tag):
+    d = f"/tmp/d32_{tag}"; shutil.rmtree(d, ignore_errors=True); os.makedirs(d)
```

Ruff (0.15.15)

[error] Probable insecure usage of temporary file or directory: "/tmp/d32_"

(S108)